### PR TITLE
[LUM-486] Added prometheus telemetry

### PR DESCRIPTION
**Pull request by Segfaultd**
### Introduction

This pull request introduces telemetry to be used in Grafana.

### Testing
No specific testing required

### CODE REVIEW

File: src/http/controllers/core.controller.ts [hunk 1]
```diff
@@ -157,4 +167,25 @@ export class CoreController {
...
+        } else if (data.name == MetricNames.LUM_PRICE_EUR) {
+            await this._lumPriceEUR.set(data.value);
+        } else if (data.name == MetricNames.DFRACT_MA_BALANCE) {
+            await this._dfrMaBalance.set(data.value);
+        } else if (data.name == MetricNames.MARKET_CAP) {
+            await this._marketCap.set(data.value);
+        } else if (data.name == MetricNames.TWITTER_FOLLOWERS) {
+            await this._twitterFollowers.set(data.value);
+        }
+    }
```
Comment (Ricardo-Remy) [inline src/http/controllers/core.controller.ts:190]: Would it make it make a bit more readable to do it in the object literal way ?

```
async updateMetric = @Payload() data: { name: string; value: number }): Promise<void> =>
    ({
        [MetricNames.LUM_CURRENT_SUPPLY]: await this._lumCurrentSupply.set(data.value),
        [MetricNames.DFRACT_CURRENT_SUPPLY]: await this._dfrCurrentSupply.set(data.value),
        [MetricNames.COMMUNITY_POOL_SUPPLY]: await this._communityPoolSupply.set(data.value),
        ...
    }[data[name] || defaultValue);
```

**Comment by Ricardo-Remy** [comment]
Overall, there are some synergies that could be done together with 439 in the scheduler part.

File: src/async/schedulers/metric.scheduler.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import { LumNetworkService } from '@app/services';
+
+@Injectable()
+export class MetricScheduler {
+    constructor(@Inject('API') private readonly _client: ClientProxy, private readonly _lumNetworkService: LumNetworkService) {}
+
+    @Cron(CronExpression.EVERY_5_SECONDS)
+    async update() {
+        // Acquire data
+        const [dfrSupply, lumSupply] = await Promise.all([this._lumNetworkService.client.queryClient.bank.supplyOf('udfr'), this._lumNetworkService.client.queryClient.bank.supplyOf('ulum')]);
```
[the remark this answers is not in the document]
Comment (Segfaultd) [reply src/async/schedulers/metric.scheduler.ts:15]: Agree. Lets see after 439 is merged.

File: src/async/schedulers/metric.scheduler.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        const [communityPool] = await Promise.all([this._lumNetworkService.client.queryClient.distribution.communityPool()]);
+        const [dfrBalance] = await Promise.all([this._lumNetworkService.client.queryClient.dfract.getAccountBalance()]);
```
[the remark this answers is not in the document]
Comment (Segfaultd) [reply src/async/schedulers/metric.scheduler.ts:17]: Agree. Lets see after 439 is merged.

File: src/http/controllers/core.controller.ts (hunk 1 above)
Comment (Segfaultd) [reply src/http/controllers/core.controller.ts:190]: Not possible using NestJS with the MessagePattern syntax

**Pull request merged**